scripts/activate.sh

```diff
@@ -13,26 +13,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [[ -z "${CUDA_HOME}" ]]
+if [[ -z "${OPEN_CE_CUDA_HOME}" ]]
 then
-  echo "CUDA_HOME is not set. Checking common places.."
+  echo "OPEN_CE_CUDA_HOME is not set. Checking common places.."
 
   if [[ -d "/usr/local/cuda" ]]
   then
      echo "/usr/local/cuda found."
-     export CUDA_HOME=/usr/local/cuda
+     export OPEN_CE_CUDA_HOME=/usr/local/cuda
   elif [[ -d "/usr/local/cuda-10.2" ]]
   then
      echo "/usr/local/cuda-10.2 found."
-     export CUDA_HOME=/usr/local/cuda-10.2
+     export OPEN_CE_CUDA_HOME=/usr/local/cuda-10.2
```

Comment:
Should the cuda version come from conda_build_config.yaml like {CUDATOOLKIT} instead of being hard-coded?

Reply:
Issue #2 is intended to address this.

Reply:
yeah, let;s leave that to a separate work effort.